lib/recipe_api_wrapper.rb

```diff
@@ -0,0 +1,38 @@
+require 'httparty'
+
+class RecipeApiWrapper
+
+  URL = "https://api.edamam.com/search"
+  RPARAM = "http%3A%2F%2Fwww.edamam.com%2Fontologies%2Fedamam.owl%23recipe_"
+  ID = ENV["APP_ID"]
+  KEY = ENV["APP_KEY"]
+
+  @recipes_list = []
+
+  def self.find_recipes(search_terms)
+
+    if !@recipes_list.empty?
+      @recipes_list = []
+    end
+
+    encoded_search_terms = URI.encode(search_terms)
```

Comment:
There's no checking to ensure that `search_terms` is a string and it's non-empty.